main.c

```diff
@@ -109,17 +113,20 @@ void setup() {
 #ifdef MPU6050_ENABLED
     mpu6050_init();
 #ifdef MPU6050_SELFTEST
-    if (mpu6050_selfTest()) {
+    retcode = mpu6050_selfTest(); //store retcode from selfTest function
+    if (retcode) {                //check retcode
 #ifdef MAVLINK_EN
         mavlink_message_t msg;
 
+        sprintf(msg_str,"MPU6050 SELFTEST FAILED. CODE: %0d",
+        retcode);                 //create c-string with retcode
```

Comment:
Please, use space ' ' after comma ','. It improves code readability.
As an example: `foo(a, b, c)`

In addition, if you want to use multi-line function call then please use separated lines for each argument.
As an example: `foo(a, b, c)` to
```
foo(a,
    b,
    c);
```